doc/Language/quoting.pod6

```diff
@@ -525,6 +525,38 @@ option {
 };
 =end code
 
+Some other situations to pay attention to are innocent-looking ones
+where the text looks like a Raku expression. For example, the
+following generates an error:
+
+=begin code :lang<text>
+my $title = 'USAFA Class of 1965';
+say qq:to/HERE/;
+<a href='https://usafa-1965.org'>$title</a>
+HERE
+# Output:
+Type Str does not support associative indexing.
+  in block <unit> at here.raku line 2
+=end code
+
+The right angle bracket makes '$title' look like a hash call
+to Raku when it is actually a Str variable, hence the error message.
```

Comment:
Nit: `C<Str>` Maybe also clarify you refer to `$title</a>`. And maybe something instead of hash "call". Hash index?